x-pack/plugin/eql/qa/common/src/main/java/org/elasticsearch/test/eql/CommonEqlRestTestCase.java

```diff
@@ -43,14 +48,9 @@
             {"{\"query\": \"" + validQuery + "\", \"filter\": {}}", "query malformed, empty clause found"}
     };
 
-    @BeforeClass
-    public static void checkForSnapshot() {
-        assumeTrue("Only works on snapshot builds for now", Build.CURRENT.isSnapshot());
```

Comment:
This check is being performed in other places in code. If we remove this one here, should it be removed anywhere else as well?

Reply:
There are several improvements that can be made to this class but I wanted to keep it separate:
1. the class / index name needs to be changed
2. the snapshot was suppose to be removed outside the feature flag - it hasn't so maybe there are cases for this.
3. The class should be refactored so it contains only the loading and utility tests, not any tests itself.